### DISCUSSION TITLE
show array in reverse order ... just like telegram or whatsapp

### DISCUSSION THREAD

**Discussion by hr-sargazi** [oleggrishechkin/react-viewport-list#70] [Q&A]
hi, i am trying to create a view for my conversation view with your package , but when i use css attributes like 

transform: scaleY(-1);
and 
flex-direction: column-reverse;

the view would not show the array row's correctly as if it shows just one row of the array 

i hope you could guide me to have some solution for this issue .

steps to reproduce :

open project's sandbox link

https://codesandbox.io/s/react-viewport-list-xw2rt

and insert those two css attributes to the list's container

**Comment by oleggrishechkin** [comment]
Hi, @shazoak 
You can't use css hacks like flex reverse and this type of transform. 

You need to manually reverse items array to render. Rendered items order must be equal to DOM order, otherwise virtualization will be incorrect.
